oaknational/Oak-Web-Application
# LESQ-732: Utilise new API on Unit Listing page

## Description

Music year: 1960

- Updates Unit Listing page to point at new API
- Remove theme title from Unit cards
- Creates Unit Listing fixture data

## Issue(s)

Fixes #LESQ-732

## How to test

1. Go to https://deploy-preview-2356--oak-web-application.netlify.thenational.academy
2. Click on any unit listing page, both legacy and new content should display

## Screenshots

How it used to look (delete if n/a):
![Screenshot 2024-04-12 at 10 55 54](https://github.com/oaknational/Oak-Web-Application/assets/91190841/cbe8dc13-90c5-4196-a7aa-bb3288c8a33d)

How it should now look:
![Screenshot 2024-04-12 at 10 55 18](https://github.com/oaknational/Oak-Web-Application/assets/91190841/cd5d1e0d-31a9-4e4d-b500-d21d84432642)
![Screenshot 2024-04-12 at 10 59 31](https://github.com/oaknational/Oak-Web-Application/assets/91190841/d7ae8f7a-bc69-4d89-95b2-34769e0d49a0)

## Checklist
